examples/rigid/debug.py

```diff
@@ -0,0 +1,434 @@
+import argparse
+import os
+import numpy as np
+import time
+from PIL import Image
+import genesis as gs
+from typing import List
+import imageio
+import cv2
+import pickle
+import torch
+from genesis.engine.solvers.rigid.rigid_solver_decomp import RigidSolver
+
+
+def save_2D_to_video(list_frames: List, filepath, fps: int = 60):
+
```

Comment:
Avoid skipping line after `def`, `if`...etc This is not part of standard Python formatting conventions.